### ISSUE TITLE
Read link (href) address - keyboard shortcut to read URL from the anchors href

### ISSUE THREAD

**Issue by michelhebert** [nvaccess/nvda#6595]
This feature doesn't seem to be listed anywhere in the guide. I'm unsure of the various implementations NVDA covers, but I'm looking at this from an HTML perspective (web pages). Also would be a help with the [WCAG 2.0 guideline 2.4 Navigable](https://www.w3.org/WAI/WCAG20/quickref/#navigation-mechanisms).

I've noticed NVDA doesn't give the user the option of retrieving the URL when focusing on an anchor element. Considering the various ways anchors are used, this would help the user get context on what the link is going to send them (or not).

e.g.

`<a href="http://www.nvaccess.org">Use a screen-reader</a>`

NVDA would read "http://www.nvaccess.org" 

Considering the various functionality and abuse of anchor tags on the web, I believe this feature allows visually impaired users to know (and decide) if they want to ignore a link when it is misleading or an action they do not want:

href="#"
href="javascript:void(0)"
href="mailto:...."
href="file:///..."
etc..

OSX's VoiceOver has [this feature](https://www.apple.com/voiceover/info/guide/_1131.html#vo27972), which is quite nice.
Reference link: https://www.apple.com/voiceover/info/guide/_1131.html#vo27972

Maybe set the keyboard shorcuts as follows:
- **Link URL**: Once on link, press **k, k** (successively) to read link address
- **Visited link**: Press "u" to get to link, then **u, u** (successively) to read visited link address.
- **Unvisited link**: Press "v" to get to link, then **v, v** (successively) to read unvisited link address.

note: Pressing k,k (successively) also would allow a user that uses the Tab key to get to a link, the ability to just get the link address (regardless of link state - visited/unvisited).

Thanks!

**Comment by jcsteh** [comment]
I don't follow why a screen reader user is any different to any other user
in this regard. A URL is essentially an implementation detail. We shouldn't
expect screen reader users to rely on this any more than any other user.
Furthermore, browsers provide a way to access the URL of a link via the
context menu. Thus, if a user does want this, they can get it in the same
way that a sighted user might.

**Comment by feerrenrut** [comment]
In chrome at least there is a way for sighted users to be able easily see the URL of a link. When hovering over a link, the URL is shown in the status bar. I tend to use this quite a bit when I'm not sure where a link might take me.

**Comment by jcsteh** [comment]
It's possible that the report status bar command doesn't work correctly in
Chrome. If that's true, that should be fixed. That would make for an
equivalent experience.

**Comment by feerrenrut** [comment]
It should be possible to use the `report status bar` command to see the URL of the link (however there is a bug raised against this, see #6597). It is also possible to use the context menu to get the URL, by selecting `copy link address` and then using the `report clipboard` command (with `NVDA+C`). We believe this addresses this issue.
